build.gradle.kts

```diff
@@ -167,6 +167,7 @@ fun getTags(inputFilePath: String): List<Pair<String, String>> {
     val regex = """<!ENTITY\s+(.+)\s+"([^"]+)">""".toRegex()
     return file(inputFilePath).bufferedReader().lines()
         .filter { it.matches(regex) }
+        .distinct()
```

Comment:
Needed because the current JMdict_e.xml has TWO lines with the same tag:
```xml
<!ENTITY ik "word containing irregular kana usage">
```
which was causing an error to throw.